### DISCUSSION TITLE
Rec TV

### DISCUSSION THREAD

**Discussion by BahJohn** [iptv-org/community#727] [Q&A]
Channel Name: Rec TV (Canal 13)
Channel ID: RecTV.cl
Country: Chile
Language: Spanish
Website: https://www.13.cl/rec
Stream URL (optional):
Notes (optional): xxx

**Comment by AntiPontifex** [comment]
Already on list: please close.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19743
